.github/workflows/ci.yml

```diff
@@ -45,7 +45,8 @@ jobs:
       - run: yarn nx affected --target=type-check
       - run: yarn nx affected --target=lint
       - run: yarn nx affected --target=test
-      - run: yarn nx affected --target=component-test --skipInstall
+      - run: npx playwright clear-cache
```

Comment:
why is this needed ?

Reply:
ignore this, I was checking why it's still failing, as locally it was fine and only on CI failing. I guess I can close this PR, let's use yours with the fix.

Reply:
kk sounds good. ty